Mobile/starter_project_dark_knights/lib/features/user_profile/presentation/screen/profile_page.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:flutter/material.dart';
+
+
+import '../../../../core/utils/colors.dart';
+import '../widgets/blueCard.dart';
+import '../widgets/post_display.dart';
+
+
+
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
This should only be a part of your main.dart there should only be one MaterialApp.

Reply:
Done